simpletransformers/experimental/classification/classification_model.py

```diff
@@ -308,6 +308,7 @@ def train(self, train_dataset, output_dir, show_running_loss=True, eval_df=None)
                         else:
                             outputs = model(**inputs)
                         # model outputs are always tuple in pytorch-transformers (see doc)
+                    loss = outputs[0]
```

Comment:
This will solve the issue, but I think the better solution is to simply unindent 

https://github.com/ThilinaRajapakse/simpletransformers/pull/649/files#diff-1d264ccacd110fcb6a8fba220f5ecc92L317.

I think that line is where the bug is coming from.